requirements.txt

```diff
@@ -3,21 +3,21 @@ geoalchemy2==0.5.0
 psycopg2-binary==2.7.7
 pyramid==1.10.2
 pyramid-debugtoolbar==4.5
-PyYAML==5.1b1
+PyYAML==5.1b3
 shapely==1.6.4.post1  # rq.filter: ==1.6.4.post1
 simplejson==3.16.0
-SQLAlchemy==1.2.18
+SQLAlchemy==1.3.0
```

Comment:
Needs to be properly tested since this is a major release.

see: https://docs.sqlalchemy.org/en/latest/changelog/migration_13.html

Reply:
Looks fine after some testing